Anirveda.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anirveda</title>
+    <h1> Anirveda</h1>
+    <h2>The TechnoEconomic Club of PDEU</h2>
+</head>
+<body>
+    <img src="anirveda_logo.png.jpg" id="image"

```

Comment:
2. Here you are adding logo, but you haven't committed the logo image that's why logo will not be appeared here. You have to add logo and commit it, then only it will appear